ocfweb/docs/docs/staff/policies/lab-reservation-policy.md

```diff
@@ -0,0 +1,16 @@
+[[!meta title="Lab Reservation Policy"]]
```

Comment:
Should title this "Lab reservation policy (staff expectations)" or something to distinguish it from the one for members in general. Also note the casing conventions I mentioned in #340.